setup_preprocessing.py

```diff
@@ -34,6 +34,8 @@
     install_requires=required_deps,
     extras_require=extras,
     python_requires=">=3.5",
-    packages=find_packages(include=["*elasticdl_preprocessing*"],),
+    packages=find_packages(
+        include=["elasticdl_preprocessing*"], exclude=["*test*"]
```

Comment:
Why do we need to set `exclude` when we have set `include` ?

Reply:
Because elasticdl_preprocessing also contains test package.